l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -297,7 +307,7 @@ contract MailboxFacet is ZkSyncHyperchainBase, IMailbox {
         bytes[] calldata _factoryDeps,
         bytes32 _canonicalTxHash,
         uint64 _expirationTimestamp
-    ) external override returns (bytes32 canonicalTxHash) {
+    ) external override onlyL1 returns (bytes32 canonicalTxHash) {
```

Comment:
Maybe it makes sense to do the opposite as well. i.e. onlyGateway /onlyNotL1, for the functions e.g. bridgehubRequestL2TransactionOnGateway